clase 1 19.4/src/main/java/interfaces/IBoardBuilder.java

```diff
@@ -0,0 +1,22 @@
+package interfaces;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+import entities.Board;
+
+
+public interface IBoardBuilder {
+     int askUserForBoardSize();
```

Comment:
you have this method on IScanner interface too. So, I think having this method in IScanner interface does make more sense to me